chia/cmds/stop.py

```diff
@@ -49,7 +49,7 @@ async def async_stop(root_path: Path, config: Dict[str, Any], group: tuple[str,
 
 @click.command("stop", help="Stop services")
 @click.option("-d", "--daemon", is_flag=True, type=bool, help="Stop daemon")
-@click.argument("group", type=click.Choice(list(all_groups())), nargs=-1, required=True)
+@click.argument("group", type=click.Choice([g for g in list(all_groups()) if g != "daemon"]), nargs=-1, required=True)
```

Comment:
Why shouldn't we be able to stop the daemon?

Reply:
Because there is already `-d/--daemon` flag.
Besides, when services like fullNode, farmer are running, stopping only a daemon results in stopping all the services connecting to the daemon.  so `chia stop daemon` is the same as `chia stop -d all`.